src/Table/TableBody.js

```diff
@@ -30,6 +30,11 @@ class TableBody extends Component {
      */
     displayRowCheckbox: PropTypes.bool,
     /**
+     * Controls if the border is set on a tbody.
```

Comment:
Please could you use `TableBody` to refer to the component here and anywhere else it appears?